docs/howtos/ensembling.rst

```diff
@@ -230,7 +230,7 @@ Next we transform the ``train_epoch`` function.
     batch_metrics = []
     for perm in perms:
       batch = {k: v[perm, ...] for k, v in train_ds.items()}
-      batch = jax_utils.replicate(batch) #!
+      batch = jax.device_put_replicated(batch, jax.local_devices()) #!
```

Comment:
It is a bit annoying that we now have to add `jax.local_devices()` everywhere... I will check with the JAX team whether it makes sense to set `devices` to `jax.local_devices()` by default we so we can simply omit it everywhere.

Reply:
Even if the JAX team does agree to set `devices` to `jax.local_devices()` by default, wouldn't that be a different PR because from what I can tell `jax.local_devices()` is used everywhere?